.meteor/packages

```diff
@@ -27,9 +27,11 @@ sacha:spin
 multiply:iron-router-progress
 zeroasterisk:throttle
 percolate:synced-cron
-accounts-facebook
 service-configuration
 yang2007chun:materialize-scss
 digilord:faker
+meteorhacks:kadira-debug
```

Comment:
btw should be no references to kadira-debug as there is no code added to make use of it.